Application/EdFi.Ods.Api/Security/Authorization/PersonResourceGraphAuthorizationTransformer.cs

```diff
@@ -141,6 +159,17 @@ private static void ApplyContactTransformation(BidirectionalGraph<Resource, Asso
                 resourceGraph.RemoveEdge(directContactDependency);
                 resourceGraph.AddEdge(new AssociationViewEdge(studentContactAssociationResource, directContactDependency.Target, directContactDependency.AssociationView));
             }
+
+            string LogAndThrowException()
```

Comment:
This was separated out into separate method not for reuse, but because the containing structure uses pattern matching and you can only provide an expression here (not a block). Because I wanted to expand what was being done, I was forced to move it to a separate method.